<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Attendance</class>
 <widget class="QWidget" name="Attendance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="back_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Trở lại</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="info_line">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>21</y>
     <width>1171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>Khi bạn sẵn sàng hãy ấn vào nút bắt đầu để điểm danh</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="preview_widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>640</width>
     <height>640</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QPushButton" name="attend_button">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>540</y>
      <width>140</width>
      <height>70</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Điểm danh</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="cam_widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-1</y>
      <width>640</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="recognition_widget" native="true">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>110</y>
     <width>640</width>
     <height>320</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="tittle">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>601</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Trích xuất khuôn mặt điểm danh</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>641</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="face_preview">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="attendance_info" native="true">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>430</y>
     <width>640</width>
     <height>320</height>
    </rect>
   </property>
   <widget class="QTableWidget" name="attendance_table">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>71</y>
      <width>641</width>
      <height>231</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string>MSSV</string>
     </property>
     <property name="font">
      <font>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tên sinh viên</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Email</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Thời gian</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="tittle_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>601</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Thông tin sinh viên đã điểm danh</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
